keyboards/sepsepkbd/yubino_hara/readme.md

```diff
@@ -0,0 +1,32 @@
+# yubino_hara
+
+![yubino_hara](https://raw.githubusercontent.com/nagmeal/sep-sep-kbd/main/yubino-hara/yubinohara.jpg)
+
+- Split
+- Reverse tilt (tilted from front to back)
+- Symmetrical staggered
+
+
+
+* Keyboard Maintainer: [t-ando](https://github.com/nagmeal)
+* Hardware Supported: *YUBINO-HARA PCB, RP2040-zero*
+* Hardware Availability: *[PCB & Case Data](https://github.com/nagmeal/sep-sep-kbd/tree/main/yubino-hara)*
+
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb sepsepkbd/yubino_hara -km default
+
+Flashing example for this keyboard:
+
+    Connect to PC with USB cable and copy the generated fa2 file.
```

Comment:
```suggestion
    make sepsepkbd/yubino_hara:default

Flashing example for this keyboard:

    make sepsepkbd/yubino_hara:default:flash
```

Reply:
Oops, I should not have changed this.
Thanks. I will fix it.